deno-installer.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+set -u
+
+main() {
+
+  need_cmd downloader --check
+  need_cmd uname
+  need_cmd mktemp
+  need_cmd chmod
+  need_cmd mkdir
+  need_cmd rm
+  need_cmd rmdir
+  need_cmd jq
+
+  get_architecture || return 1
+
+  local _release_name=deno_"$RETVAL"_x64
+
+  get_release_url "$_release_name"
+
+  local _ext=".gz"
+  case "$RETVAL" in
+  *win*)
+      _ext=".zip"
+      ;;
+  esac
+
+  local _dir="$(mktemp -d 2>/dev/null || ensure mktemp -d -t deno_installer)"
+  local _file="${_dir}/deno_${RETVAL}_x64${_ext}"
+
+  ensure downloader $RELEASE_URL $_file
+
+  local _deno_home="${HOME}/deno"
+  local _deno_bin="$_deno_home/deno"
+
+  ensure mkdir -p "$_deno_home"
+
+  gunzip -c "$_file" > "$_deno_bin"
```

Comment:
This won't work for windows zip file. We could potentially change all of the packages to be zip.

Reply:
That works for me. Jest let me know if I should wait for that or do it conditionally.

Reply:
@ry I've changed it so it uses `unizp` on .zip ( windows )